EpiAware/src/EpiAwareBase/types.jl

```diff
@@ -1,17 +1,27 @@
 abstract type AbstractModel end
 
 """
-The abstract supertype for all structs that define a model for generating unobserved/latent
-infections.
+The abstract supertype for all structs that define a model for generating
+unobserved/latent infections.
 """
```

Comment:
Not really a problem, but what config are we going for on doc string cols allowed per line?

Reply:
Shall we make that a standard. I was doing 92 char per line for doc strings.